packages/core/src/components/Menu/MenuItemButton/MenuItemButton.tsx

```diff
@@ -40,7 +40,7 @@ export interface MenuItemButtonProps extends VibeComponentProps {
 
 const MenuItemButton: FC<MenuItemButtonProps> & {
   kinds?: typeof Button.kinds;
-  tooltipPositions?: typeof DialogPosition;
+  tooltipPositions?: TooltipPositionsType;
```

Comment:
same as before, you removed the static prop from here
so either get it back there, or remove line 43
```
export default withStaticProps(MenuItemButton, {
  kinds: Button.kinds,
  tooltipPositions: DialogPosition
});
```